.travis.yml

```diff
@@ -1,15 +1,27 @@
+dist: xenial
 language: ruby
-sudo: false
 rvm:
   - 1.9.3
   - 2.0.0
-  - 2.1.0
+  - 2.1.10
+  - 2.2.10
+  - 2.3.8
+  - 2.4.5
+  - 2.5.3
+  - 2.6.3
+  - jruby-9.1.17.0
   - jruby
   - rbx-2.1.1
   - ruby-head
+jdk:
+  - openjdk8 # for jruby
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: rbx-2.1.1
+env:
+  global:
+  - JRUBY_OPTS='--debug'
```

Comment:
Are you still working on this? Or do you intend to keep this flag? (and if so, why?)

Reply:
Good q. This is not needed yet unless we have coveralls on, which we don't.  Since it doesn't add value, I'll drop this. 

Oh and yes, turns out I still have some commits to add.. coming soon!